includes/class-sensei-course.php

```diff
@@ -2671,13 +2671,14 @@ public static function course_archive_sorting( $query ) {
 		$course_order_by_options = apply_filters(
 			'sensei_archive_course_order_by_options',
 			array(
+				'default' => __( 'Default sort', 'sensei-lms' ),
 				'newness' => __( 'Sort by newest first', 'sensei-lms' ),
 				'title'   => __( 'Sort by title A-Z', 'sensei-lms' ),
 			)
 		);
 
-		// setup the currently selected item
-		$selected = 'newness';
+		// setup the currently selected item.
+		$selected = 'default';
```

Comment:
I wonder if it should always be `default`, or if it should be `default` only when we have the option `sensei_course_order`. Otherwise, we should the default, but we sort like the `newness`.
WDYT?

Reply:
Yeah I thought about displaying the `default` option only if the course order was set, but I thought that it would complicate the experience a little so I decided to simplify it and show it always.
When no course order is set the behaviour will be as you described: sorting by newness even though default is selected. 
In the end "default" is quite ambiguous so we can use it on our benefit 😄